airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/S3StorageOperations.java

```diff
@@ -127,15 +127,16 @@ public String uploadRecordsToBucket(final SerializableBuffer recordsData,
       }
 
       try {
-        return loadDataIntoBucket(objectPath, recordsData);
+        final String fileName = loadDataIntoBucket(objectPath, recordsData);
+        LOGGER.info("Successfully loaded records to stage {} with {} attempt(s)", objectPath, exceptionsThrown.size());
```

Comment:
is `exceptionsThrown` zero indexed? like if there were no exceptions thrown and a successful sync happened on the first try would this be `with 0 attempt(s)` ?

Reply:
Good catch, I'll refactor the string to more accurately describe this as `0 reattempt(s)`. As for capturing _some_ data the path we're looking to go forward with is to use a profiler that Subodh has implemented and/or the datadog agent used on the Sources team to observe the time to complete a serialization of `AirbyteRecordMessage`s